src/admin/tables/task.php

```diff
@@ -27,257 +27,4 @@ public function __construct(&$db)
 		parent::__construct('#__admintask_tasks', 'id', $db);
```

Comment:
I have asked you why you need `JObserverMapper::addObserverClassToClass` this line of code? Do you explain me why you have used this?